features/favorite.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Favorite
+
+  Scenario: Favorite Creation
+    Given I am logged into the site
+    When I visit a user's profile
+    And I click "Follow"
+    When I visit the "/users/following" page
```

Comment:
Found it. It's actually [still in a PR](https://github.com/smashingboxes/guides/pull/3), but [here's the guide](https://github.com/smashingboxes/guides/blob/feature/cucumber-guide/qa/cucumber.md).